client/task-list/tasks/PaymentGatewaySuggestions/components/Setup/Setup.js

```diff
@@ -151,17 +151,13 @@ export const Setup = ( {
 		isPaymentGatewayResolving ||
 		! isPluginLoaded;
 
-	const DefaultStepper = useCallback(
-		( props ) => (
-			<Stepper
-				isVertical
-				isPending={ stepperPending }
-				currentStep={ needsPluginInstall ? 'install' : 'configure' }
-				steps={ [ installStep, configureStep ].filter( Boolean ) }
-				{ ...props }
-			/>
-		),
-		[ stepperPending, installStep, configureStep ]
+	const defaultStepper = (
```

Comment:
Hey @louwie17 & @joshuatf , this change made it so we can't override props for the `<DefaultStepper />` in the slotfill, which I thought was kind of a cool feature. Was that intentional and still desired?